### PR TITLE
Update mailbox-usage.md

### LINKED ISSUE #12249: The storage chart is calculated based on what exactly?
Hello, 

Question about the documentation that is not precise and which I wish that you adjust for the public for better understanding of the charts:
The Mailbox Usage > Storage chart is calculated based on TotalItemSize or (TotalItemSize + TotalDeletedItemSize) in Get-MailboxStatistics? are the Recoverable Items included?
image

It will be really helpful to know what is really included there.

Thank you in advance.
Best regards

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e612a4e4-d051-27a7-e54a-bb1ac3fe0849
* Version Independent ID: 653be32f-e4e8-8a72-98b3-a629238ecbd5
* Content: [Microsoft 365 admin center mailbox usage reports - Microsoft 365 admin](https://learn.microsoft.com/en-US/microsoft-365/admin/activity-reports/mailbox-usage?WT.mc_id=365AdminCSH_inproduct&view=o365-worldwide)
* Content Source: [microsoft-365/admin/activity-reports/mailbox-usage.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/activity-reports/mailbox-usage.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
added a statement about recoverable items

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12249

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/activity-reports/mailbox-usage.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ To access shared mailbox information, change the drop-down selection at the top
 
 The **Mailbox usage** report can be viewed for trends over the last 7 days, 30 days, 90 days, or 180 days. However, if you select a particular day in the report, the table will show data for up to 28 days from the current date (not the date the report was generated). The data in each report usually covers up to the last 24 to 48 hours.
 
+The **Mailbox usage** report does not include  **Recoverable Items** as they are included in **Recoverable Items mailbox quota**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/mailbox-usage.md]:
```suggestion
The **Mailbox usage** report does not include **Recoverable Items** as they are included in the **Recoverable Items mailbox quota**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**